src/Pages/LoginPage/LoginPage.jsx

```diff
@@ -61,10 +61,11 @@ export const LoginPage = ({ setToken }) => {
         localStorage.setItem("token", JSON.stringify(result.token));
         setToken(JSON.stringify(result.token));
         var messages = "Login success";
-        Notification(messages);
+        Notification(messages, "success");
         navigate("/");
       } catch (error) {
         console.log(error);
```

Comment:
remove log

Reply:
thankyou for review, this issue already fix on 4da229e93e8066f1e4b3e33ea8e2a40e32f9c6d0